config.toml

```
@@ -33,5 +33,10 @@ name = "<i class='fab fa-discourse'></i> Discussion Forums"
 url = "https://discuss.libp2p.io"
 weight = 4
 
+[[menu.shortcuts]]
+name = "<i class='fab fa-discourse'></i> Connectivity"
```

```suggestion
name = "<i class='fab fa-book'></i> Connectivity"
```

It's not a discourse page, I suggest we use book.
A better choice is browser: https://fontawesome.com/icons/browser?s=solid&f=classic but I don't know what fontawesome version this site uses